controller.py

```diff
@@ -333,8 +444,9 @@ def updates_feed(self, collection_details):
                 precomposed_entries.append(entry)
             else:
                 # There is no cached OPDS entry. We'll create one later.
-                works_for_feed.append((work, identifier))
+                works_for_feed.append((identifier, work))
```

Comment:
(work, identifier) is the wrong order. This error wasn't caught because in all of the test cases, there was already a cached OPDS entry for the work.